wodles/aws/aws_s3.py

```diff
@@ -1053,6 +1061,12 @@ def iter_files_in_bucket(self, aws_account_id=None, aws_region=None):
                         debug(f"+++ Remove file from S3 Bucket:{bucket_file['Key']}", 2)
                         self.client.delete_object(Bucket=self.bucket, Key=bucket_file['Key'])
                     self.mark_complete(aws_account_id, aws_region, bucket_file)
+                    processed_logs += 1
+
+                # This is a workaround in order to work with custom buckets that don't have
+                # base prefix to search the logs
+                if processed_logs == 0:
+                    self._print_not_logs_to_prcess_message(self.bucket, aws_account_id, aws_region)
```

Comment:
Typo in the method's call
```suggestion
                    self._print_not_logs_to_process_message(self.bucket, aws_account_id, aws_region)
```